### DISCUSSION TITLE
Please please a J2ME version?

### DISCUSSION THREAD

**Discussion by mrc2rules** [BombusMod/BombusMod#92] [Q&A] [answered]
Hello! Is there a still working J2ME bombus client? I've been looking for one, and the only one i found is named 2go. If you guys have one that still works, please let me know. A lot of us are eager to have a client ☺️

**Comment by vitalyster** [comment]
j2me version was in Ovi/Opera Store, that store may still work, try this? http://ovi.sigma.apps.bemobi.com/en_us/bombusmod_bombusmod_n1.html?pos=1

**Reply by vitalyster** [reply]
If that works you can also try to install my copy from [this link](https://bombusmod.github.io/BombusMod/BombusMod.jad) and let me know does it work too?

**Reply by mrc2rules** [reply]
Hmm, got a jar? I have problems with jad

**Reply by vitalyster** [reply]
Which problem? Please share the full error message. Which link? From store or from GitHub? Please give me as much info as possible

**Reply by mrc2rules** [reply]
I mean, I can't install over jad files, to install jad versions i need to directly visit the website. And github doesn't load there obv. And the ovi website doesn't load in my android idk why. Maybe ISP blocks it. Thats why I'm asking gor a jar if possible. I'm trying to transfer it via Bluetooth to my phone. My phone is Nokia Asha 310 btw. Let me know if u have discord

**Reply by vitalyster** [reply]
You can try [JAR](https://bombusmod.github.io/BombusMod/BombusMod.jar) too, but that means it will not be signed and will ask for permissions for internet and filesystem as usual. I need to test signed install but does not have any working j2me phone :)

**Reply by vitalyster** [reply]
If you will able to transfer both JAR and JAD to the phone from Github URL, then please try to install from JAD too, it should try to install JAR located in the same directory on the phone and that install should be signed/trusted if JAD is correct.

**Reply by mrc2rules** [reply]
> You can try [JAR](https://bombusmod.github.io/BombusMod/BombusMod.jar) too, but that means it will not be signed and will ask for permissions for internet and filesystem as usual. I need to test signed install but does not have any working j2me phone :)

I can help you with that! I have a j2me phone:), permission thing not a problem for me and hopefully its touch compat lol as my asha is a touch phone

**Reply by mrc2rules** [reply]
> If you will able to transfer both JAR and JAD to the phone from Github URL, then please try to install from JAD too, it should try to install JAR located in the same directory on the phone and that install should be signed/trusted if JAD is correct.

Sorry for the delay, I'll get back to you soon.

**Reply by vitalyster** [reply]
Don't worry, other people already tested this and everything works as expected

**Comment by vitalyster** [comment, marked as answer]
[JAD, signed](http://bombusmod.github.io/BombusMod/BombusMod.jad)
[JAR](http://bombusmod.github.io/BombusMod/BombusMod.jar)

**Reply by JoseskVolpe** [reply]
JAD gave me "Unable to verify AP files" on LG x350 after downloading JAR. Installing directly from JAR worked

**Reply by vitalyster** [reply]
JAD from Bemobi store, JAD from Github or both?

**Reply by JoseskVolpe** [reply]
Github

**Reply by vitalyster** [reply]
Does it work from Bemobi?

**Reply by JoseskVolpe** [reply]
No

**Reply by vitalyster** [reply]
OK, thanks. People report it works correctly with Nokia and SonyEricsson. LG was never a target device/brand for BombusMod :)

**Reply by JoseskVolpe** [reply]
¡your welcome!

**Reply by mrc2rules** [reply]
I can't seem to get past the sign up page, how do i sign up? For example my number is 0501234567 and country code +966, and what is JID?

**Reply by vitalyster** [reply]
Are you serious? JID is provided by your XMPP service. I thought you know what XMPP is :)

**Reply by vitalyster** [reply]
You can try to register a new account on our jabber.ru service by clicking "register new account" if you have no idea what XMPP service to use

**Reply by mrc2rules** [reply]
I actually discovered XMPP while searching for j2me chat clients, i do enter everything but i get to a blank page and I don't get any SMS code... A

**Reply by vitalyster** [reply]
So if registration inside the app is not working for some reason then just register an account on any XMPP service website and enter provided JID into the app

**Reply by mrc2rules** [reply]
And just a little suggestion, make the buttons a bit bigger for touchscreens, they are tiny

**Reply by vitalyster** [reply]
You can adjust size of buttons in settings

**Reply by mo-krauti** [reply]
From which version of the source code was the j2me version `0.8.1440.1348` built? I could not find a matching git tag

**Reply by vitalyster** [reply]
1348 is a revision number in the old svn repo, use 0.8.9999 tag for latest version with netbeans j2me project files

**Reply by mo-krauti** [reply]
Thank you, what is the preprocessor being used? 
Does not seem to be [java-comment-preprocessor](https://github.com/raydac/java-comment-preprocessor) or [jpp](https://github.com/abego/jpp)

**Reply by vitalyster** [reply]
@mo-pyy if you want to discuss code-related questions please create a new discussion.
It uses Netbeans preprocessor which was a part of Netbeans Mobility Pack. There were attempts to use Antenna preprocessor and somehow it was possible to build Android target using Antenna. All preprocessor instructions are removed after 0.8.9999.

**Reply by donnyworks** [reply]
doesn't really work on my blackberry sadly
from what i can tell it's something about the zlib compression?

**Reply by vitalyster** [reply]
@donnyworks first of all you need properly configured XMPP server, which server you are using?
